docs/src/docs/examples.clj

```diff
@@ -12,11 +12,12 @@
 
 ;; tag::start-standalone-node[]
 (defn start-standalone-node ^crux.api.ICruxAPI [storage-dir]
-  (crux/start-node {:crux.node/topology '[crux.standalone/topology]
```

Comment:
so the vector form doesn't conform the actual spec

Reply:
This was bought in by something currently awaiting release - https://github.com/juxt/crux/pull/589

Reply:
Thanks! Yeah, I forgot that docs could point to unreleased changes.
#610 would be awesome